README.md

```diff
@@ -14,14 +14,14 @@ before you run all the tests. If not set, the tests that use these variables wil
 
 ## S3 Source
 The S3 Source Connector connects to a S3 bucket with the provided configurations, using
-`aws.bucket`, `aws.access-key-id`,`aws.secret-access-key` and `aws.region`. Then will
+`aws.bucket`, `aws.accessKeyId`,`aws.secretAccessKey` and `aws.region`. Then will
```

Comment:
Maybe it's good enough if we say "aws.accessKey"? IIUC, the S3 docs also refer to them as "access keys".